ci/test.sh

```diff
@@ -13,8 +13,9 @@ chown -R ubuntu:ubuntu /cache/
 echo '127.0.1.11 internal-stargate.local' >> /etc/hosts
 echo '127.0.1.12 internal-stargate.local' >> /etc/hosts
 
-# restart docker
-service docker --full-restart
+# Ensure we can connect to Docker daemon to resolve issue #1580, see:
+# https://dhananjay4058.medium.com/solving-docker-permission-denied-while-trying-to-connect-to-the-docker-daemon-socket-2e53cccffbaa
+usermod -a -G docker ubuntu
```

Comment:
hmm adding a user to that group without restarting the docker service seems odd.. @tatu-at-datastax what was this motivated by? was my fix not working?

and bu the way this was already done in the image building: https://github.com/stargate/stargate/blob/5f6381da305a1f63725fea7df6f68df993eb70ab/ci/Dockerfile#L38-L42

so this is surely not the fix for the issue!

Reply:
Ok, I can undo it then. The fix did not seem to have any effect; I had 2 out of 3 failures after fix went in.

Reply:
Sorry about this, I was wondering if user/group permissions were set but did not actually check.